lib/runtime/src/lib.rs

```diff
@@ -143,11 +154,47 @@ pub fn compile(wasm: &[u8]) -> error::CompileResult<Module> {
 /// `error::CompileError`, `error::LinkError`, or
 /// `error::RuntimeError` (all combined into an `error::Error`),
 /// depending on the cause of the failure.
-#[cfg(feature = "wasmer-clif-backend")]
+#[cfg(feature = "default-compiler")]
 pub fn instantiate(wasm: &[u8], import_object: &ImportObject) -> error::Result<Instance> {
     let module = compile(wasm)?;
     module.instantiate(import_object)
 }
 
+/// Compile wasm into a [`Cache`] that can be stored to a file or
```

Comment:
Maybe we should only export this if cache is enabled?

Reply:
I swear I did feature-gate this, will fix.